modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/search/test/CalendarBookingIndexerLocalizedContentTest.java

```diff
@@ -111,28 +113,25 @@ public void testJapaneseTitleFullWordOnly() throws Exception {
 		String description = StringUtil.toLowerCase(
 			RandomTestUtil.randomString());
 
-		Stream.of(
-			full, partial1, partial2
-		).forEach(
-			title -> addCalendarBooking(
+		List<String> titles = new ArrayList<>(
+			Arrays.asList(full, partial1, partial2));
+
+		LocalizedValuesMap descriptionMap = new LocalizedValuesMap() {
```

Comment:
@jeffwu0724 I would prefer the original way, although it's worse in performance. Usually we prefer clear code in tests than optimized-for-performance code.